client/src/components/LayoutComponents/Form/index.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { Fragment, Component } from 'react';
+import {
+	Checkbox,
+	FormControlLabel,
+	Grid,
+	TextField,
+	Select,
+	MenuItem,
+	InputLabel,
+	Button,
+	FormControl,
+} from '@material-ui/core';
+import axios from 'axios';
+import Alert from '@material-ui/lab/Alert';
+import Dialog from '../../CommonComponents/Dialog';
+
+class Form extends Component {
+	state = {
+		questions: [],
+		newQuestionText: '',
+		newQuestionType: '',
+		loading: true,
+		error: '',
+		typeError: '',
+		selectedQuestions: {},
+		submitDialogOpen: false,
+	};
+
+	componentDidMount = () => {
+		axios
+			.get('/dashboard/form')
+			.then(({ data }) => {
+				if (!data)
+					return this.setState({
+						loading: false,
+						error: 'There was an error loading your data',
+					});
+				this.setState({ questions: data, loading: false });
+			})
+			.catch((err) =>
+				this.setState({
+					error: 'There was an error loading your data',
+				}),
+			);
+	};
+
+	handleCheckQuestion = (question) => {
+		this.setState({
+			selectedQuestions: {
+				...this.state.selectedQuestions,
+				[question.id]: !this.state.selectedQuestions[question.id],
+			},
+		});
+	};
+
+	handleChangeNewQuestionText = (event) => {
+		let { value } = event.target;
```

Comment:
This is not being reassigned. Use const. Eslint should tell you about it. Do you have it running?

Reply:
I do have it running